decidim-core/lib/decidim/exporters/csv.rb

```diff
@@ -16,13 +16,13 @@ class CSV < Exporter
       #
       # Returns an ExportData instance.
       def export
-        data = ::CSV.generate(headers: headers, write_headers: true, col_sep: ";") do |csv|
+        data = ::CSV.generate(headers: headers, write_headers: true, col_sep: "\t") do |csv|
```

Comment:
Separated with tabs? O_o'

Reply:
Excel is freaking weird, I'm looking at an alternate solution.